src/ucp/tag/offload.c

```diff
@@ -403,6 +403,30 @@ static ucs_status_t ucp_tag_offload_eager_zcopy(uct_pending_req_t *self)
                                     ucp_proto_am_zcopy_req_complete);
 }
 
+static size_t ucp_tag_offload_rndv_pack_rkey(ucp_request_t *sreq, ucp_lane_index_t lane,
```

Comment:
Looks like no need to introduce an extra function. You may use the same for regular and offload RNDV.

Reply:
rndv pack will be changed later